userspace/falco/falco_metrics.cpp

```diff
@@ -222,10 +222,16 @@ std::string falco_metrics::to_text(const falco::app::state& state)
 		{
 			prometheus_metrics_converter.convert_metric_to_unit_convention(metric);
 			std::string namespace_name = "scap";
-			if (metric.flags & METRICS_V2_RESOURCE_UTILIZATION || metric.flags & METRICS_V2_KERNEL_COUNTERS)
+			if (metric.flags & METRICS_V2_RESOURCE_UTILIZATION || metric.flags & METRICS_V2_KERNEL_COUNTERS || metric.flags & METRICS_V2_PLUGINS)
```

Comment:
We can now remove `metric.flags & METRICS_V2_PLUGINS` here, but it's also not an issue.